en/news/_posts/2015-12-25-ruby-2-3-0-released.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: news_post
+title: "Ruby 2.3.0 Released"
+author: "naruse"
+translator:
+date: 2015-12-25 00:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.3.0.
+
+This is the first stable release of Ruby 2.3 series.
+It introduces many new features and improvements for example:
+
+A [Frozen String Literal Pragma](https://bugs.ruby-lang.org/issues/11473)
+is introduced.
+With Ruby 2.1, `"str".freeze` has been optimized to reduce object allocation.
+Ruby 2.3 introduces a new magic comment and command line option to
+freeze all string literals in the source files.
+Additionally for debugging, you can get where the object is created on
+`"can't modify frozen String"` error by
+`--debug=frozen-string-literal` command line option.
+
+A [safe navigation operator](https://bugs.ruby-lang.org/issues/11537)
+([so-called lonely operator](https://instagram.com/p/-M9l6mRPLR/)) `&.`,
+which already exists in C#, Groovy, and Swift, is introduced to ease
+`nil` handling as `obj&.foo`. `Array#dig` and `Hash#dig` are also added.
+Note that this behaves as [try! of ActiveSupport](http://apidock.com/rails/Object/try%21),
```

Comment:
Consider link `try!` to official Ruby on Rails API document:

http://api.rubyonrails.org/v4.2.5/classes/Object.html#method-i-try-21

instead of 

http://apidock.com/rails/Object/try%21